### ISSUE TITLE
Timeouts on first start or after long idle

### ISSUE THREAD

**Issue by Esshahn** [ether/etherpad-lite#5129]
**Describe the bug**
Whenever Etherpad restarts or after long idle times, loading times are slow and throws errors. Only after several browser refreshes the page comes up.

**To Reproduce**
1. Etherpad Lite deployed on render.com
2. postgres db deployed on render.com
3. after new deploys or after long idle times, Etherpad loading times are very long, including timeouts

**Screenshots**
<img width="826" alt="bildschirmfoto_2021-07-26_um_10 59 11" src="https://user-images.githubusercontent.com/434355/126969649-df032799-bd76-442f-b582-b5387c0b80e9.png">

````
Jul 26 10:59:00 AM  [2021-07-26 08:59:00.846] [INFO] Minify - Compress JS file /opt/etherpad-lite/src/static/js/contentcollector.js.
Jul 26 10:59:01 AM  [2021-07-26 08:59:01.553] [INFO] Minify - Compress JS file /opt/etherpad-lite/src/static/js/Changeset.js.
Jul 26 10:59:01 AM  [2021-07-26 08:59:01.747] [WARN] client - TimeoutError: The operation timed out. -- {
Jul 26 10:59:01 AM    errorId: 'cG4MsHcAbkNvj9HwPzDl',
Jul 26 10:59:01 AM    type: 'Unhandled Promise rejection',
Jul 26 10:59:01 AM    msg: 'TimeoutError: The operation timed out.',
Jul 26 10:59:01 AM    url: 'https://XXXXX',
Jul 26 10:59:01 AM    source: 'unknown',
Jul 26 10:59:01 AM    linenumber: -1,
Jul 26 10:59:01 AM    userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.3 Safari/605.1.15',
Jul 26 10:59:01 AM    stack: 'send@[native code]\n' +
Jul 26 10:59:01 AM      'M@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:2303\n' +
Jul 26 10:59:01 AM      'X@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:2896\n' +
Jul 26 10:59:01 AM      'k@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:3670\n' +
Jul 26 10:59:01 AM      'i@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:4296\n' +
Jul 26 10:59:01 AM      'L@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:4317\n' +
Jul 26 10:59:01 AM      'https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:5035\n' +
Jul 26 10:59:01 AM      'H@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:5081\n' +
Jul 26 10:59:01 AM      'https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:5487\n' +
Jul 26 10:59:01 AM      'G@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:5504\n' +
Jul 26 10:59:01 AM      'e@https://pad.digital-vereint.berlin/static/js/require-kernel.js?v=439ac1b6:10:6355\n' +
Jul 26 10:59:01 AM      'https://pad.digital-vereint.berlin/javascripts/lib/ep_etherpad-lite/static/js/pad.js?callback=require.define&v=439ac1b6:49:4461\n' +
Jul 26 10:59:01 AM      'asyncFunctionResume@[native code]\n' +
Jul 26 10:59:01 AM      '[native code]\n' +
Jul 26 10:59:01 AM      'promiseReactionJobWithoutPromise@[native code]\n' +
Jul 26 10:59:01 AM      'promiseReactionJob@[native code]'
Jul 26 10:59:01 AM  }
````

**Comment by webzwo0i** [comment]
This looks like a tmeout when doing HTTP GET for the packaged code. First start should regenerate the package files, however long idle should not regenerate them but read them from file system.

So it looks like:
- the instance is running on a more or less slow CPU
- timeouts should be increased
- there is a problem with Minifiy/caching_middleware that results in the file getting compressed multiple times, although it should be read from fs. I thought I fixed a similar issue long time ago so are you sure your running latest release?

**Comment by Esshahn** [comment]
Thank you for your quick reply @webzwo0i , very appreciated :)

> the instance is running on a more or less slow CPU

That might be correct, it's a shared CPU with 512MB.
Stats look okay to me though:

<img width="1003" alt="Bildschirmfoto 2021-07-26 um 12 48 14" src="https://user-images.githubusercontent.com/434355/126977535-adf3505a-1c2d-454e-b05b-880bd7f43033.png">

<img width="1026" alt="Bildschirmfoto 2021-07-26 um 12 48 06" src="https://user-images.githubusercontent.com/434355/126977558-dd15d56f-1a0f-45bf-a02b-fa8ceb58aab0.png">

> timeouts should be increased

can you point me to where I can change that?

> there is a problem with Minifiy/caching_middleware that results in the file getting compressed multiple times, although it should be read from fs. I thought I fixed a similar issue long time ago so are you sure your running latest release?

That's right, I see that twice each time the machine boots up.
I'm using the develop branch from about a week ago.

**Comment by rhansen** [comment]
Is `dbType` set to `postgres` or `postgrespool` in your `settings.json`? If `postgres`, please try `postgrespool` to see if that improves anything.

**Comment by Esshahn** [comment]
thanks @rhansen , I did indeed change the type to `postgrespool` a couple of days ago and have not seen any timeouts since. I recommend documenting this setting. Thanks for the support!

**Comment by rhansen** [comment]
Fixed in PR #5163.
